docs/src/examples.md

```diff
@@ -2,543 +2,180 @@
 
 ## 1D Column examples
 
-## 2D Cartesian examples
```

Comment:
Hi @Snowdog85123 , thank you for updating the file and posting the screenshots of the rendered documentation. You should have kept everything that was in the original section under the header [`2D Cartesian examples`](https://clima.github.io/ClimaCore.jl/dev/examples/#2D-Cartesian-examples) in the original file though. Now, it seems like you are only including your contributions and you accidentally deleted the existing parts. Thank you